express/features/carousel/carousel.js

```diff
@@ -0,0 +1,193 @@
+import { getLibs } from '../../scripts/utils.js'
+const { createTag ,  loadStyle } = await import(`${getLibs()}/utils/utils.js`);
+
+function correctCenterAlignment(plat) {
```

Comment:
I like where you've put the carousel.js file, but before adding files I'd recommend checking to to see if this logic or file has already been added :)  
If you do you'll see we already have a carousel.js file.